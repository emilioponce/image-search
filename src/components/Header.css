.Wrapper {
  background-color: #202329;
  color: white;
}

.Header {
  display: flex;
  flex-direction: row;
  align-items: baseline;
  padding: 10px;
}

.Title {
  font-size: 160%;
  text-transform: uppercase;
  color: #61dafb;
  letter-spacing: 0.02em;
}

.Subtitle {
  font-size: 115%;
  margin-left: 10px;
}
