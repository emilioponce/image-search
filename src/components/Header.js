import React, { Component } from 'react'
import './Header.css'

const Header = () => {
  return (
    <div className="Wrapper">
      <div className="Header">
        <div className="Title">Image Search</div>
        <div className="Subtitle">
          A Flickr Search Gallery using react & redux
        </div>
      </div>
    </div>
  )
}

export default Header
