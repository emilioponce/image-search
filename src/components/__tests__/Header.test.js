import React from 'react'
import ReactDOM from 'react-dom'

import { shallow } from 'enzyme'
import { configure } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'

import Header from '../Header'

configure({ adapter: new Adapter() })

const component = <Header />

describe('Header component', () => {
  it('Renders without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(component, div)
    ReactDOM.unmountComponentAtNode(div)
  })

  it('Snapshot matchs', () => {
    const wrapper = shallow(component)
    expect(wrapper).toMatchSnapshot()
  })
})
