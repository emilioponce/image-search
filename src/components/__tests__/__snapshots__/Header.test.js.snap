// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component Snapshot matchs 1`] = `
<div
  className="Wrapper"
>
  <div
    className="Header"
  >
    <div
      className="Title"
    >
      Image Search
    </div>
    <div
      className="Subtitle"
    >
      A Flickr Search Gallery using react & redux
    </div>
  </div>
</div>
`;
